src/models/InstrumentModel.js

```diff
@@ -0,0 +1,125 @@
+import {
+  EntityModel,
+} from './index'
+
+import _ from 'lodash'
+
+const ACCOUNTS_LIST_RESOURCE = '/v1/accounts'
```

Comment:
@chris-revops Lets make an INSTRUMENTS_LIST_RESOURCE method instead of a constant that duplicates code from the Accounts model.

```
const INSTRUMENTS_LIST_RESOURCE = (account_id) => `/v1/accounts/${account_id}`
```